linux_os/guide/system/auditing/auditd_configure_rules/audit_privileged_commands/audit_rules_privileged_commands_kmod/rule.yml

```diff
@@ -60,3 +68,4 @@ template:
     name: audit_rules_privileged_commands
     vars:
         path: /usr/bin/kmod
+        path@ubuntu2004: /bin/kmod
```

Comment:
@cipherboy can you check if this path is correct? I looked in the ubuntu STIG, but I haven't tested it.

Reply:
These can be changed back if needed.

Reply:
Both paths exist, due to usr merge.